### ISSUE TITLE
Spelling page for languages > 30 characters alphabet

### ISSUE THREAD

**Issue by willwade** [AceCentre/SpeakBook#30]
For languages such as Polish  - the character set is quite big. 

We need a way of showing all these characters on the fingerspelling page..

My suggestion is that we create a template for these languages where we put the fingerspelling page into Landscape - and allow for a greater number of keys for spelling. 

We don't want this as a default layout - it has a downside that it increases the range of movement to select letters. But it feels the right compromise for big alphabet languages.

**Comment by willwade** [comment]
see https://github.com/AceCentre/SpeakBook/commit/69a27013887a5a11fedc3b86e4e94f46055fe9d0 - attempted this for croatian https://speakbook.netlify.app/hr/speakbook/classic.html

**Comment by willwade** [comment]
Duplication of #42

**Comment by hosseinamin** [comment]
Sorry for the dumb question, But which page is using for finger-spelling? I guess it is the last page.

**Comment by willwade** [comment]
Yes! Last page! The one that has a kind of keyboard on it. See also #42
